### PR TITLE
usage example added in extract glimpse

### PR DESCRIPTION
**Pull request by imransalam**
Added usage example in extract glimpse. Issue raised on https://github.com/tensorflow/tensorflow/issues/30097

### CODE REVIEW

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -3678,6 +3678,24 @@ def extract_glimpse(
 
   Returns:
     A `Tensor` of type `float32`.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```
Comment (rthadur) [inline tensorflow/python/ops/image_ops_impl.py]: please remove >> symbol

Comment (imransalam) [reply tensorflow/python/ops/image_ops_impl.py]: @rthadur , Hey I removed the >> symbol. Thanks

Comment (rthadur) [reply tensorflow/python/ops/image_ops_impl.py]: thank you

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -3678,6 +3678,22 @@ def extract_glimpse(
...
+    ```python
+    import tensorflow as tf
+    BATCH_SIZE = 2
+    IMAGE_HEIGHT = 256
+    IMAGE_WIDTH = 256
+    CHANNELS = 3
+    GLIMPSE_SIZE = (24, 24)
+    image = tf.random.normal(shape=(BATCH_SIZE, IMAGE_HEIGHT, 
+    IMAGE_WIDTH, CHANNELS) )
+    offsets_variable = tf.random.uniform(shape=(BATCH_SIZE, 2))
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: Why offsets_variable? Why not just offsets?

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -3678,6 +3678,22 @@ def extract_glimpse(
 
   Returns:
     A `Tensor` of type `float32`.
+    
+  Usage Example:
+    ```python
+    import tensorflow as tf
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: No need to import tensorflow

File: tensorflow/python/ops/image_ops_impl.py [hunk 4]
```diff
@@ -3678,6 +3678,22 @@ def extract_glimpse(
...
+    IMAGE_HEIGHT = 256
+    IMAGE_WIDTH = 256
+    CHANNELS = 3
+    GLIMPSE_SIZE = (24, 24)
+    image = tf.random.normal(shape=(BATCH_SIZE, IMAGE_HEIGHT, 
+    IMAGE_WIDTH, CHANNELS) )
+    offsets_variable = tf.random.uniform(shape=(BATCH_SIZE, 2))
+    output = tf.image.extract_glimpse(image, size=GLIMPSE_SIZE, 
+    offsets=offsets_variable)
+    print(output.shape)
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: It'd be nice if we could show something here at all. If I read this code and not execute it I get almost no information out of it (other than the requirements on the input shapes of the op).

Can we pick a known small one-channel image and show what a glimpse is?

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 4 above)
Comment (imransalam) [reply tensorflow/python/ops/image_ops_impl.py]: Sure let me add that

File: tensorflow/python/ops/image_ops_impl.py [hunk 5]
```diff
@@ -3681,17 +3681,24 @@ def extract_glimpse(
...
+    GLIMPSE_SIZE = (3, 3)
+    image = tf.random.uniform(shape=(BATCH_SIZE, IMAGE_HEIGHT, 
+      IMAGE_WIDTH, CHANNELS) )
     output = tf.image.extract_glimpse(image, size=GLIMPSE_SIZE, 
-    offsets=offsets)
-    print(output.shape)
+      offsets=[[1, 1]], centered=False, normalized=False)
+    print(image)
+        <tf.Tensor: id=172, shape=(3, 3), dtype=float32, numpy=
+        array([[0.98461604, 0.14595735, 0.3296578 ],
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: Showing particular random numbers is not super helpful. Can you put numbers like 1. 2. 3. etc in the image so a reader has an idea of where each input ended up?

Comment (imransalam) [reply tensorflow/python/ops/image_ops_impl.py]: Hey I've made the changes, let me know if it's fine or do we need more changes?

**Review by alextp — APPROVED** [review]

**Pull request merged**